lib/rust/ensogl/core/src/debug/monitor.rs

```diff
@@ -829,16 +790,15 @@ impl Sampler for Fps {
 // ==================
 
 /// Sampler measuring the memory usage of the WebAssembly part of the program.
-#[derive(Clone, Copy, Debug, Default)]
+#[derive(Debug, Default)]
 pub struct WasmMemory {
-    value:       f64,
-    value_check: ValueCheck,
+    stats: Stats,
 }
```

Comment:
Why this structure is defined by hand instead of using macro, as almost all other structures?